.github/workflows/ci.yml

```diff
@@ -214,8 +217,9 @@ jobs:
           .
           --wait
           --create-namespace
-          --set controller.image.repository=$(echo ${{ steps.meta.outputs.tags }} | cut -d ":" -f 1)
-          --set controller.image.tag=$(echo ${{ steps.meta.outputs.tags }} | cut -d ":" -f 2)
+          --set nginxGateway.image.repository=$(echo ${{ steps.meta.outputs.tags }} | cut -d ":" -f 1)
+          --set nginxGateway.image.tag=$(echo ${{ steps.meta.outputs.tags }} | cut -d ":" -f 2)
+          --set nginxGateway.image.pullPolicy=Never
```

Comment:
These tests were failing to deploy things properly, and it was because we were trying to Always pull from a registry, where this image doesn't exist. This was broken and we didn't realize it because it would just be pulling our edge image, not the image under test.